### ISSUE TITLE
Deutsche Übersetzung in Betreff (beim Versenden eines Datensatzes per Mail)

### ISSUE THREAD

**Issue by coeh** [ubtue/tuefind#3078]
Nicht Folge des Upgrades, sondern auch auf dem Livesystem so - ist mir aber jetzt beim Testen aufgefallen:

Wenn ich im Nutzerkonto eingeloggt bin, die Sprache auf Deutsch steht und ich einen Datensatz per mail verschicke, lautet der Betreff "Referenz aus IxTheo: [Titel]"
M.E. ist "Referenz" ein false friend als (in diesem Kontext nicht zutreffende) deutsche Übersetzung von reference.

Kann man das umbenennen? Und was wäre eine gute deutsche Übersetzung:
- "Literaturangabe"
- "Titel"
- noch etwas anderes?

**Comment by coeh** [comment]
heute in der FR-Runde diskutiert. Bitte Betreff im Deutschen ändern zu "Suchergebnis aus IxTheo".

**Comment by relhei** [comment]
Bitte auch analog für RelBib umsetzen. Und vmtl. auch KrimDok?

**Comment by coeh** [comment]
Oh Entschuldigung, Mareike - ich bin davon ausgegangen, dass das (da kein Label an diesem Eintrag) für alle Systeme gilt. Das passt aber natürlich nicht dazu, dass ich oben "aus IxTheo" geschrieben habe. Sorry, betriebsblind.

Sollen wir es in KrimDok genauso machen, oder ist da etwas anders, @JMeister01 ?

**Comment by JMeister01** [comment]
Gerne. Danke!

**Comment by coeh** [comment]
An #2625 hatte ich mich gar nicht mehr erinnert ;) Das ist aber, wenn ich recht sehe, inhaltlich komplett analog. Das in #953 gesehene Problem bezog sich ja auf einen anderen Teil des gleichen Betreffs. Heute morgen in der FR-Runde bestand Einigkeit, dass der Begriff Referenz unglücklich ist.

Zur eigentlichen Frage:
Wir haben auch darüber gesprochen, ob sich das auf beide Fälle (Suchergebnisliste und Kurztitelanzeige) beziehen soll. Ergebnis: Ja! Einer der Gründe für den Begriff "Suchergebnis" war, dass er nicht festlegt, ob es sich um eine Einzahl oder Mehrzahl von Titeln handelt und der Betreff somit in allen vorkommenden Fällen gleich lauten kann.

**Comment by mtrojan-ub** [comment]
Bitte direkt im Translation Tool korrigieren. (Findet man relativ einfach wenn man dort nach "Referenz" sucht und im Target Scope "VuFind Translations" auswählt.)

**Comment by coeh** [comment]
> > Alles klar, vielen Dank. Ich habe leider keinen Zugriff aufs Translationtool. [@derclou](https://github.com/derclou) kannst du dich darum kümmern?
> 
> Liebe [@coeh](https://github.com/coeh), tut mir leid, aber ich habe keinen Zugriff auf die Spalte für Deutsch, nur auf Englisch und Französisch.

Ah okay - wer hat denn Zugriff auf die Spalte für Deutsch? @socheres ?

Inzwischen kommt mir allerdings noch eine andere Frage: Haben wir den Begriff reference nur im spezifischen Kontext dieser Mails? Denn es wäre ja auch wieder sehr problematisch "reference" immer und in jedem Kontext mit "Suchergebnis" zu übersetzen. In diesem konkreten Fall ist aus dem Kontext heraus Suchergebnis gemeint, bzw. das ist das, womit man auf Deutsch die in dieser Mail angesprochene Sache ausdrücken kann, damit sich jemand das Richtige darunter vorstellt. In anderen Kontexten würde "Literaturangabe" "Zitation" etc. besser  passen - diese Begriffe passen wiederum aber hier nicht... D.h. ist das Translationtool hier das geeignete Werkzeug? Kann man da überhaupt je nach Kontext unterschiedliche deutsche Übersetzungen für das gleiche englische Wort eingeben?

Oder wäre die Lösung, dass wir in diesem konkreten Fall auch das Englische in den Mails mit anpassen und im englischsprachigen Betreff search result schreiben müssen, damit wir einen kontextspezifischen Begriff haben, der dann auch eindeutig zu übersetzen ist? Mir wäre das in der Sache fast am sympathischsten, aber ich kenne natürlich die technische Seite nicht.

**Comment by thefass** [comment]
Berechtigungen fürs Deutsche und überhaupt für Dich, @coeh,  könnte @jriedl einrichten. 
Im Translationtooll finde ich den Einzelbegriff "Referenz" aber gar nicht. vielleicht mache ich was falsch?

**Comment by derclou** [comment]
> Berechtigungen fürs Deutsche und überhaupt für Dich, [@coeh](https://github.com/coeh), könnte [@jriedl](https://github.com/jriedl) einrichten. Im Translationtooll finde ich den Einzelbegriff "Referenz" aber gar nicht. vielleicht mache ich was falsch?

Doch, wenn man den Begriff "Referenz" unter den VuFind Translations sucht, findet man den token, um den es geht (bulk_email_title). Vielleicht hast du unter den Keywords gesucht?

**Comment by coeh** [comment]
Alles klar, vielen Dank! Ich schließe.
